tests/app/celery/test_process_delivery_status_result_tasks.py

```diff
@@ -536,3 +543,225 @@ def test_get_include_payload_status_exception(notify_api, mocker, sample_notific
         'app.celery.process_delivery_status_result_tasks.dao_get_callback_include_payload_status', side_effect=exception
     )
     assert not _get_include_payload_status(sample_notification())
+
+
+def test_ut_can_retry_sms_request_valid_case_returns_true():
```

Comment:
suggestion: this and the next three test could be done with [parametrization ](https://docs.pytest.org/en/7.1.x/example/parametrize.html#parametrizing-tests)